ds_judgements_public_ui/sass/includes/_document_navigation_links.scss

```diff
@@ -27,7 +27,7 @@
       top: 0;
       background-color: $color__white;
       box-sizing: border-box;
-      width: 100%;
+      width: 100vw;
```

Comment:
These feel like a very different PR?

Reply:
Oops, sorry, a rebase SNAFU - there's already another PR open for that stuff and I accidentally opened this branch on top of that one. I'll fix before merging this one 😊